app.js

```diff
@@ -2,6 +2,8 @@ const express = require('express');
 const app = express();
 const port = process.env.PORT || 3000;
 const multer = require('multer');
+const mongoose = require('mongoose');
```

Comment:
You can delete this line of code to prevent a merge conflict since my topic is about Mongoose, I installed and connected it already in my own branch. Only need to push this asap to develop branch.

Reply:
@pepijnkok  Code in a branch should always work on its own and not be dependent on another branch to function. If this would result in a merge conflict, we would fix that when it happens

Reply:
@daoneandonly Yeah I know,  just letting jeroen know that there will be a conflict and how to easily fix it.